lunatrace/bsl/backend/src/github/login.ts

```diff
@@ -97,12 +53,12 @@ function getGithubOrgIdsFromApiResponse(userId: string, userGithubOrgs: GetUserO
   ];
 }
 
-export const githubLogin = async (req: Request, res: Response): Promise<void> => {
+export async function githubLogin(req: Request, res: Response): Promise<void> {
   // todo: fix this unsafe property access
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+   
   const userId: string = req.body.ctx.identity.id as string;
 
-  console.log(`[user: ${userId}] Github login webhook started`);
+  log.info(`[user: ${userId}] Github login webhook started`);
```

Comment:
I think I might see a cleaner way to use this logger rather then passing it the same value over and over as part of the string. I'm going to go play with it

Reply:
I reset my changes because I couldnt get this logger to do what I want. We can revisit later maybe